CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,39 @@
 
 import Foundation
 
-print("Hello, World!")
+func generateLotteryNumber() -> Set<Int> {
+    var lotteryNumbers: Set<Int> = []
+
+    while lotteryNumbers.count < 6 {
+        lotteryNumbers.insert(Int.random(in: 1...45))
+    }
+
+    return lotteryNumbers
+}
+
+func compareLotteryNumber(myNumbers: Array<Int>, lotteryNumbers: Set<Int>) {
+    let bothMineAndLottery = lotteryNumbers.intersection(myNumbers).sorted()
+    
+    if bothMineAndLottery.isEmpty {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    } else {
+        print("축하합니다! 겹치는 번호는", terminator: " ")
+        printAllElementsOf(bothMineAndLottery)
+        print("입니다!")
+    }
+}
+
+func printAllElementsOf(_ array: Array<Int>) {
+    for (index, number) in array.enumerated() {
+        if index == array.count-1 {
+            print(number, terminator: " ")
+        } else {
+            print(number, terminator: ", ")
+        }
+    }
+}
+
+let myLottoNumbers: [Int] = [1,2,3,4,5,6]
```

Comment:
일반적으로 콤마 다음은 한칸 띄우는게 기본 컨벤션입니다!
저는 이게 더 보기 편하던데.. 사바사니까 강요하지는 않을게요! 😅
```suggestion
let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
```